geopandas/io/arrow.py

```diff
@@ -235,6 +235,15 @@ def _validate_metadata(metadata):
         if column_metadata["encoding"] != "WKB":
             raise ValueError("Only WKB geometry encoding is supported")
 
+        if column_metadata.get("edges", "planar") == "spherical":
+            warnings.warn(
+                f"The geo metadata indicate that column '{col}' has spherical edges, "
+                "but GeoPandas ignores this metadata and will interpret the edges of "
+                "the geometries as planar.",
```

Comment:
```suggestion
                f"The geo metadata indicate that column '{col}' has spherical edges, "
                "but because GeoPandas currently does not support spherical "
                "geometry, it ignores this metadata and will interpret the edges of "
                "the geometries as planar.",
```

I think it is worth including _why_ we do that.